src/constants.telemetry.ts

```diff
@@ -466,3 +466,5 @@ type SubscriptionEventData = {
 		Record<`previous.subscription.${string}`, string | number | boolean | undefined> &
 		Record<`previous.subscription.previewTrial.${string}`, string | number | boolean | undefined>
 >;
+
+export type TrackingContext = 'graph' | 'launchpad' | 'visual_file_history' | 'worktrees';
```

Comment:
This is being called `TrackingContext` now, and we moved it to the telemetry constants, but it's for more than just telemetry. Sending a context key to the login route actually changes the content that is shown on the landing page.

For example, here is the landing page for `https://gitkraken.dev/login?source=gitlens&context=graph`:

![image](https://github.com/user-attachments/assets/1d59e7cb-1bba-48c7-b013-b7d8d557235b)